dashboard/app/views/levels/editors/_droplet.html.haml

```diff
@@ -1,34 +1,42 @@
-.field
-  = f.label :edit_code,'Droplet level'
-.field
+%legend.control-legend{data: {toggle: "collapse", target: "#dropletPalette"}}
+  Starting Block Palette (Droplet)
+
+#dropletPalette.collapse
   = f.label :code_functions, 'Edit Droplet Palette'
   = f.text_area :code_functions, placeholder: 'Droplet Palette', rows: 4, value: @level.code_functions ? JSON.pretty_generate(@level.code_functions) : ''
 
-.field
+  .field
+    = f.label :palette_category_at_start
+    = f.select :palette_category_at_start, options_for_select(@level.class.palette_categories, @level.palette_category_at_start), {include_blank: true}
+
+  .field
+    = render partial: 'levels/editors/checkboxes', locals: {f: f, field_name: :droplet_tooltips_disabled, description: "Hide droplet documentation tooltips"}
+  .field
+    = render partial: 'levels/editors/checkboxes', locals: {f: f, field_name: :lock_zero_param_functions, description: "Don't allow parameters to be added to user-defined functions with no parameters"}
+
+%legend.control-legend{data: {toggle: "collapse", target: "#starterCode"}}
+  Starter Code
+
+#starterCode.collapse
   = f.label :start_blocks, 'Edit Starting JavaScript Program'
   .row
     .span8
       ~ f.text_area :start_blocks, placeholder: 'Start Blocks', rows: 4, value: @level.uses_droplet? ? @level.start_blocks : @level.pretty_block('start')
```

Comment:
TIL about [`~` in HAML](http://haml.info/docs/yardoc/file.REFERENCE.html#whitespace_preservation)!

Reply:
me too!